packages/create-gatsby/src/index.ts

```diff
@@ -71,6 +108,18 @@ export async function run(): Promise<void> {
   console.log(`Let's answer some questions:\n`)
   const data = await prompt<IAnswers>(questions)
 
+  const selectedPlugins = [data.cms, data.styling, ...data.features].filter(
```

Comment:
I'm already creating an array of selected plugins below, so you could use that instead